code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -44,7 +44,9 @@
 	if(user.faction == team)
 		user << "You can't move your own flag!"
 		return
-
+	if(loc == user)
+		if(!user.unEquip(src))
```

Comment:
Actually, if you're just checking instead of actually dropping, we have a `canUnEquip(item)` proc.